egs/csj/s5/cmd.sh

```diff
@@ -10,6 +10,12 @@
 # conf/queue.conf in http://kaldi-asr.org/doc/queue.html for more information,
 # or search for the string 'default_config' in utils/queue.pl or utils/slurm.pl.
 
-export train_cmd="queue.pl --mem 2G"
```

Comment:
It's safer to leave train_cmd as queue.pl.  If you leave it as run.pl, there's a danger that people run it using run.pl on a cluster, and overload nodes.  At least if it's queue.pl, it fails immediately if you have no queue.